.github/workflows/pytest.yml

```diff
@@ -36,7 +36,6 @@ jobs:
           pip install -e .
           if [ -f pip/cpu_requirements.txt ]; then pip install -r pip/cpu_requirements.txt; fi
           if [ -f pip/dev_requirements.txt ]; then pip install -r pip/dev_requirements.txt; fi
-          if [ -f pip/ci_requirements.txt ]; then pip install -r pip/ci_requirements.txt; fi
```

Comment:
I removed `ci_requirements.txt` and merged everything into `dev_requirements.txt`. I can also merge the `readthedocs_requirements.txt` too if preferred